closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -63,10 +65,14 @@ compile-libs :
 	export \
 			$(OPENJCEPLUS_VS_LIB) \
 			GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
-			JAVA_HOME=$(OPENJCEPLUS_BOOT_JDK) \
+			JAVA_HOME=$(OPENJCEPLUS_JDK) \
 			JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
 			PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
-		&& $(MAKE) -j1 -C $(OPENJCEPLUS_JGSKIT_MAKE_PATH) -f $(OPENJCEPLUS_JGSKIT_MAKE) all
+		&& $(MAKE) \
+			-C $(OPENJCEPLUS_JGSKIT_MAKE_PATH) \
+			-f $(OPENJCEPLUS_JGSKIT_MAKE) \
+			OPENJCEPLUS_HEADER_FILES=$(OPENJCEPLUS_HEADER_FILES) \
+			noheaders
```

Comment:
While `noheaders` looks like it will work, I don't think the `all` target is reliable. Instead of a separate target, I suggest the OpenJCEPlus makefiles should conditionally handle generation of header files: this command would end with something like `OPENJCEPLUS_BUILD_HEADERS=false all` (the default would assume `OPENJCEPLUS_BUILD_HEADERS=true` which would use `javac` to ensure the header files exist).

Reply:
The `all` target would only fail in this particular scenario where someone is trying to bundle. I'm not sure a use case where someone wouldn't have to compile the headers, or try to compile them with a different java version compared to the rest of the classes, exists. I'm worried that we will be adding conditionals for a very particular scenario that we only encounter when bundling with `Semeru`. I think separate targets is more straightforward and doesn't unnecessarily complicate the makefile.